src/components/Search/SearchDropdown.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import SearchViewHelper from '../../utils/searchViewHelper';
+
+const { handleChangeDropdown } = SearchViewHelper;
+
+function SearchInput(props) {
```

Comment:
A few comments explaining the goal for this component as well as how it's supposed to be used would be useful for a future reader.